src/testdir/test_filetype.vim

```diff
@@ -823,6 +823,7 @@ def s:GetFilenameChecks(): dict<list<string>>
     yaml: ['file.yaml', 'file.yml', 'file.eyaml', 'any/.bundle/config', '.clangd', '.clang-format', '.clang-tidy', 'file.mplstyle', 'yarn.lock'],
     yang: ['file.yang'],
     yuck: ['file.yuck'],
+    zathurarc: ['zathurarc'],
```

Comment:
These keys are sorted alphabetically, so this should be after `z8a`

Reply:
Thanks, fixed.